src/databases/functions/users/index.js

```diff
@@ -6,6 +6,7 @@ const getUserFollowerList = require('./get-user-follower-list');
 const getUserByUserId = require('./get-user-by-user-id');
 const userBlockByAdmin = require('./user-block-by-admin');
 const userUnblockByAdmin = require('./user-unblock-by-admin');
+const setWPUUser = require('./set-w-pu-user');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new module `setWPUUser` is imported but there's no context provided about what it does. Make sure to add comments or documentation for this function in its definition file (`./set-w-pu-user`) to ensure maintainability and readability of the code.

```diff
+ // setWPUUser updates a user's score in the database
  const setWPUUser = require('./set-w-pu-user');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->